packages/next/cli/next-start.ts

```diff
@@ -58,6 +59,11 @@ const nextStart: cliCommand = (argv) => {
     port = 0
   }
 
+  if (!existsSync(`${process.cwd()}/.next/BUILD_ID`)) {
+    const { nextBuild }  = await import("./next-build");
+    await nextBuild(args as string[])
```

Comment:
we need `await` in order for the server to start after the build finishes or else it errors.